src/modules/counting.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-plusplus */
```

Comment:
[OPTIONAL] Please do not disable linter rules, this defeats the purpose of the rules themselves, which is to standardize the code and keep it clean. There is a reason for every rule. In this case, `length++` can be written as `length += 1` which is a clearer syntax of what's going on.